tests/Core/Util/Common/PrepareForOutputTest.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Tests for the \PHP_CodeSniffer\Util\Common::prepareForOutput() method.
+ *
+ * @author  Bartosz Dziewoński <dziewonski@fastmail.fm>
+ * @license https://github.com/PHPCSStandards/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ */
+
+namespace PHP_CodeSniffer\Tests\Core\Util\Common;
+
+use PHP_CodeSniffer\Util\Common;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Tests for the \PHP_CodeSniffer\Util\Common::prepareForOutput() method.
+ *
+ * @covers \PHP_CodeSniffer\Util\Common::prepareForOutput
+ */
```

Comment:
Done.

FWIW, on my Windows device the `phpunit` tests take less than 30 seconds.

Reply:
> FWIW, on my Windows device the `phpunit` tests take less than 30 seconds.

That's good to know. I spend some time on this in January and brought the test suite runtime on Windows (at the time) down from over 8 minutes to 30 seconds, so that clearly is still having effect.

For CI, it's mostly that I'm concerned about the code coverage runs, especially on PHP 5.4. That already takes something like 20 minutes now on Linux and I fear on Windows it will be much higher.

Last time I did a complete code coverage run locally on Windows (~1,5 years ago) it took over 20 hours on PHP 8.2.... and we've been adding lots and lots of tests since.